File: TicketingSystemAPI/TicketingSystemBLL/Dtos/GameDetails.cs
namespace TicketingSystemBLL.Dtos
{
    public class GameDetails
    {
        public GameDetails()
        {
            Sectors = new List<SectorHeader>();
        }
        public int Id { get; set; }

        public string HomeTeam { get; set; } = null!;
        public string AwayTeam { get; set; } = null!;
        public string HomeTeamLogoUrl { get; set; } = null!;
        public string AwayTeamLogoUrl { get; set; } = null!;
        public DateTime StartTime { get; set; }
        public string Place { get; set; } = null!;
        public virtual List<SectorHeader> Sectors { get; set; } = null!;
    }
}

File: TicketingSystem/ticketing-system/src/app/api/models/game-details.ts
/* tslint:disable */
/* eslint-disable */
import { SectorHeader } from './sector-header';
export interface GameDetails {
  awayTeam?: null | string;
  awayTeamLogoUrl?: null | string;
  homeTeam?: null | string;
  homeTeamLogoUrl?: null | string;
  id?: number;
  place?: null | string;
  sectors?: null | Array<SectorHeader>;
  startTime?: Date;
}
